zerver/webhooks/github/tests.py

```diff
@@ -193,6 +193,25 @@ def test_issue_unlabeled(self) -> None:
         expected_message = "[sbansal1999](https://github.com/sbansal1999) removed the bug label from [Issue #9](https://github.com/sbansal1999/testing-gh/issues/9)."
         self.check_webhook("issues__unlabeled", expected_topic, expected_message)
 
+    def test_issue_milestoned(self) -> None:
+        expected_topic = "testing-gh / issue #6 This is a sample issue to test GH I..."
+        expected_message = "[sbansal1999](https://github.com/sbansal1999) added the [some_random_milestone](https://github.com/sbansal1999/testing-gh/milestone/1) milestone to [issue #6](https://github.com/sbansal1999/testing-gh/issues/6)."
```

Comment:
```suggestion
        expected_message = "[sbansal1999](https://github.com/sbansal1999) added milestone [some_random_milestone](https://github.com/sbansal1999/testing-gh/milestone/1) to [issue #6](https://github.com/sbansal1999/testing-gh/issues/6)."
```

I think putting the word "milestone" before the name of the milestone will make these a bit more regular and easier to visually parse when scanning through the thread.

Reply:
(A similar remark applies to the labeled/unlabeled events. But I don't think there's any need for the wording of one to be coupled in sync with the other: we can go ahead and word this one the way we want, and then if we want to tweak the other we can separately go on to tweak it too.)

Reply:
I agree this certainly makes it easier to read.